_data-prepper/pipelines/configuration/processors/trace-peer-forwarder.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: default
+title: Trace peer forwarder
+parent: Processors
+grand_parent: Pipelines
+nav_order: 45
+---
+
+# Trace peer forwarder
+
+The `Trace peer forwarder` processor is used to reduce the number of events that are forwarded in a [Trace analytics]({{site.url}}{{site.baseurl}}/data-prepper/common-use-cases/trace-analytics/) pipeline by half when using [Peer forwarder]({{site.url}}{{site.baseurl}}/data-prepper/managing-data-prepper/peer-forwarder/). It groups the events based on `trace_id` similar to the `service_map_stateful` and `otel_trace_raw ` processors. 
+
+<!--- Why is it reducing events? Why by half? WHat is the benefit/how does it help? How does it actually reduce events? Then lead into grouping by `trace_id` --->
+
+In [Trace analytics]({{site.url}}{{site.baseurl}}/data-prepper/common-use-cases/trace-analytics/), each event is duplicated when it is sent from the `otel-trace-pipeline` to the `raw-pipeline` and to the `service-map-pipeline`. The event is forwarded once in each pipeline. When you use this processor, the event is forwarded once in the `otel-trace-pipeline` pipeline to the correct peer. 
```

Comment:
@asifsmohammed @dlvenable 

- How does this content relate to the trace peer forwarder processor? 
- How does the processor have an impact on trace peer forwarder?